### PR TITLE
Yotpo Updates 2.3.4

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Yotpo Product Reviews topics for the 2.3.4 release. 

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/marketing/yotpo-reviews-intro.html
   (added list of changes to intro topic "Changes in this release")
- https://docs.magento.com/m2/ee/user_guide/reports/review-reports.html
   (updated Reports > Review Reports to include yotpo review information)
- https://docs.magento.com/m2/ee/user_guide/reports/business-intelligence.html
   (updated Reports > Business intelligence topic to clarify analytics and insights, and to update links)

New Topic: user_guide/marketing/yotpo-reporting.html
   (Added new topic in all 3 TOCs - Yotpo reporting and Analytics - this topic is to describe the links in the Admin-reporting that previously did not link to anywhere.)

## Affected Magento editions

- [X] Open Source
- [X] Commerce
- [X] B2B

## Additional information

**staging build #28**
- user_guide/marketing/yotpo-reviews-intro.html
- user_guide/reports/review-reports.html
- user_guide/reports/business-intelligence.html
- user_guide/marketing/yotpo-reporting.html

### CODE REVIEW

File: src/reports/review-reports.md [hunk 1]
```diff
@@ -2,21 +2,26 @@
...
 
 ![]({% link images/images/reports-marketing-product-reviews.png %}){: .zoom}
-*Review Report by Product*
+_Review Report by Product_
 
 The Product Reviews Report lists all products that have been reviewed by customers. The report includes the number of reviews and average rating, the product was last reviewed, and a link to the list of reviews for each product.
+
+## Yotpo Reviews
+
+[Yotpo](https://www.yotpo.com/) is a user-generated content marketing platform enabling merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. For more Yotpo information, see [Yotpo Reviews]({% link marketing/yotpo-reviews-intro.md %}) and [Yotpo Reporting]({% link marketing/yotpo-reporting.md %}).
```
Comment (leslietilling) [inline src/reports/review-reports.md]:
```suggestion
[Yotpo](https://www.yotpo.com/) is a user-generated content marketing platform that enables merchants to gather, curate, manage, and respond to all kinds of user content from just a single platform. For more Yotpo information, see [Yotpo Reviews]({% link marketing/yotpo-reviews-intro.md %}) and [Yotpo Reporting]({% link marketing/yotpo-reporting.md %}).
```

File: src/marketing/yotpo-reviews-intro.md [hunk 2]
```diff
@@ -16,3 +16,17 @@ See the [Yotpo website](https://www.yotpo.com/platform/visual-marketing/) for mo
...
 
 With Magento v2.3.3+, integrating your Magento store is a quick, simple process and gives you access to the content and data customizations available in your Yotpo account. See [Integrate your Magento Store with your Yotpo Account]({% link marketing/yotpo-connect.md %}).
+
+## Changes In the Latest Release
+
+This release contains changes to correct a variety of issues and to update documentation and Yotpo-specific links.
+
+- The **Yotpo Reviews** and **Yotpo Analytics** links in the Admin > Reports section have been updated to redirect to the connected Yotpo account information.
+
+- Corrected issues related to the syncing of invalid orders. Invalid orders are now listed in the log, and additional sync are not attempted.
```
Comment (leslietilling) [inline src/marketing/yotpo-reviews-intro.md]:
```suggestion
- Corrected issues related to the syncing of invalid orders. Invalid orders are now listed in the log, and additional syncs are not attempted.
```

File: src/marketing/yotpo-reviews-intro.md [hunk 3]
```diff
@@ -16,3 +16,17 @@ See the [Yotpo website](https://www.yotpo.com/platform/visual-marketing/) for mo
...
+
+- Corrected an issue preventing orders from syncing correctly.
+
+- Updated _Yotpo Setup Guide_ links in the Admin.
```
Comment (leslietilling) [inline src/marketing/yotpo-reviews-intro.md]:
```suggestion
- Updated  documentation links in the Admin.
```

File: src/marketing/yotpo-reviews-intro.md [hunk 4]
```diff
@@ -16,3 +16,17 @@ See the [Yotpo website](https://www.yotpo.com/platform/visual-marketing/) for mo
...
+
+- Corrected an issue preventing Yotpo settings from displaying in the Admin after setup, when `Single-Store` mode is selected.
```
Comment (leslietilling) [inline src/marketing/yotpo-reviews-intro.md]:
```suggestion
- Corrected an issue preventing Yotpo settings from displaying in the Admin after setup when `Single-Store` mode is selected.
```

**Comment by leslietilling** [comment]
Let's run tests before merge. It is not set up at a requirement for this branch, but I think it will work.

File: src/marketing/yotpo-reviews-intro.md (hunk 4 above)
Comment (hickskenh) [reply src/marketing/yotpo-reviews-intro.md]: made change directly in file. unable to commit from suggestion

File: src/reports/review-reports.md (hunk 1 above)
Comment (hickskenh) [reply src/reports/review-reports.md]: made change directly in file. unable to commit from suggestion

**Comment by hickskenh** [comment]
running tests

**Pull request merged**